docs/developers/erc_20_integration.md

```diff
@@ -0,0 +1,82 @@
+<!--
+order: 3
+-->
+
+# ERC-20 Registration
+
+Learn how to register interoperable ERC-20s through [Evmos Governance](../users/governance/overview.md). {synopsis}
+
+## Stage 1: Drafting the ERC-20 Proposal
```

Comment:
`Stages` are not necessary if you already structure your guide with markdown levels `##`. Instead, you can guide a reader by providing an introductory text that summarized the steps, e.g.

```md
To register an erc20 follow to following steps:

1. Draft ...
2. Submit ...
3. ...

## Draft 
...
```